.github/workflows/dev.yml

```diff
@@ -10,22 +10,9 @@ on:
     - cron: "0 3 * * *"
 
 jobs:
-  cache-dependencies:
-    runs-on: ubuntu-latest
-
-    steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-node@v3
-        with:
-          node-version: lts/*
-          cache: yarn
-      - name: Install dependencies
-        run: yarn --immutable
-
+  # Let's consider the value of testing examples
+  # Perhaps we need some newer, better examples for testing
```

Comment:
I know you want to change this up but until we have time to address this, IMO we could move the neo-push-tests into its own reusable workflow. It's also used on the pull-request workflow so we also get rid of some code duplication.
Just a thought, let me know what you think.